README.md

```diff
@@ -1,2 +1,19 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) The algorithm looks for the same numbers from the list, but it must not be the same position.
+if it finds similar numbers, it returns true. Otherwise, returns that it didn't find anything.
+
+b) Because it checks the same numbers twice.
```

Comment:
This explains part (d), but does not explain why the time complexity is O(n^2): for each element of the list, the whole list is iterated over, i.e. there are two nested for loops each with n iterations